package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "foreman_theme_satellite",
-  "version": "13.1.0",
+  "version": "13.2.0",
   "description": "Satellite theme =============",
```

Comment:
Interesting description. Is this intentional?

Reply:
the formatting? yeah others have that too and I have no idea why.
given we don't upload things to the registry, the contents don't matter much anyway (besides the deps)